xbrowse_server/templates/family/family_home.html

```diff
@@ -145,12 +146,83 @@ <h4 class="section-header">Individuals</h4>
                         <td>{% if indiv.maternal_id %}{{ indiv.maternal_id }}{% else %}-{% endif %}</td>
                         <td>{% if indiv.paternal_id %}{{ indiv.paternal_id }}{% else %}-{% endif %}</td>
                         <td>{% for ipheno in indiv.get_phenotypes %}{{ ipheno.phenotype.name }}{% empty %}-{% endfor %}</td>
+                        <td><button id={{ indiv }}_{{ project.project_id }} type="button" class="btn btn-info btn-sm phenotipsEditModalBtn">Edit in Phenotips</button></td>
```

Comment:
Yes, good call,  and the way to go.. updated and committed. The project name is common to all individuals, so didn't need to be passed in with each click. Following was cleaner/simpler.

			sessionStorage.setItem('currentInFocusIndividualId',this.id)
			var patientId=this.id;
			var project_name="{{ project.project_id }}";
			showEditPageForThisPatient(patientId,project_name);